starfish/pipeline/features/pixels/__init__.py

```diff
@@ -0,0 +1,64 @@
+import argparse
+import os
+
+from starfish.io import Stack
+from starfish.pipeline.pipelinecomponent import PipelineComponent
+from starfish.util.argparse import FsExistsType
+from starfish.pipeline.features.pixels import _base
+
+
+class PixelSpotDetector(PipelineComponent):
```

Comment:
Why this name?  It's not a pixel spot that's being detected, if I understand things correctly.  Why not just SpotDetector?

Reply:
SpotDetector is a bit too general (we have a bunch of types)
I thought about this a bit. 

It could be `ConnectedPixelsSpotDetector` ... I suspect I'll have to think on this for a while.

Reply:
It's also possible we will be able to merge the `pixels` and `spots` sub-packages and move everything up into features as a result of #226